news/3 Code Health/8485.md

```diff
@@ -0,0 +1 @@
+Remove `jupyter.runMagicCommands` (which has been deprecared for over a year) in favor of `runStartupCommands`.
```

Comment:
Shouldn't we remove it from the package.json? I thought VS code did that with deprecated stuff eventually?

Reply:
It never existed in the package.json, hence the PR, it hasn't been there for a long time.